### PR TITLE
[docs] Clarify that reverse proxy setups need trusted-proxies

### PR DESCRIPTION
**Pull request by dequis**
#1125

### CODE REVIEW

File: docs/installation_guide/docker.md [hunk 1]
```diff
@@ -98,6 +98,19 @@ If you want to use [LetsEncrypt](../configuration/letsencrypt.md) for ssl certif
...
 3. (Optional) Set `GTS_LETSENCRYPT_EMAIL_ADDRESS` to a valid email address to receive certificate expiry warnings etc.
 
+#### Reverse proxies
+
+The default port bindings are for exposing GoToSocial directly and publicly. Remove the `#` in front the line that forwards `127.0.0.1:8080:8080` which makes port `8080` available only to the local host. Change that `127.0.0.1` if the reverse proxy is somewhere else.
+
+To ensure [rate limiting](../api/ratelimiting.md) by IP works, remove the `#` in front of `GTS_TRUSTED_PROXIES` and set it to the IP the requests from the reverse proxy are coming from. That's usually the value of the `Gateway` field of the docker network.
+
+```shell
+$ docker network inspect gotosocial_gotosocial | jq -r '.[0].IPAM.Config[0].Gateway'
```
Comment (tsmethurst) [inline docs/installation_guide/docker.md:108]: most people reading this likely won't have jq installed (or know what it is), so I think it's probably better to just show where in the json output you can find the gateway address.

**Comment by tsmethurst** [comment]
Thanks! Just the one comment about `jq`

File: docs/installation_guide/docker.md (hunk 1 above)
Comment (dequis) [reply docs/installation_guide/docker.md]: Reasonable!

**Comment by tsmethurst** [comment]
Grand, thank you very much! :)

**Pull request merged**